styles.css

```diff
@@ -202,6 +280,211 @@
   box-sizing: border-box;
 }
 
+.button-project768 {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .header-project {
+    transform: rotate(270deg);
+    padding-left: 20px;
+    font-weight: 600;
+    font-size: 85px;
+  }
+
+  .projects {
+    display: flex;
+    flex-direction: column;
+    padding-top: 100px;
+    background-image: url(./assests/images/image_dots.png);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    grid-area: projects;
+  }
+
+  .project {
+    grid-template-columns: 10% 10% 10% 10% 10% 10% 10% 10% 10% 10%;
+    grid-template-rows: auto 100px auto auto 100px auto auto auto auto auto;
+    grid-template-areas:
+      "projects projects projects project1 project1 project1 project1 project1 project1 project1"
+      ". . . . . . back1 back1 back1 ."
+      "project2 project2 project2 project2 . . back1 back1 back1 . "
+      "project2 project2 project2 project2 . . . . . . "
+      ". . . . . . project3 project3 project3 project3"
+      "project4 project4 project4 project4 project4 .  project3 project3 project3 project3  "
+      "project4 project4 project4 project4 project4 . back2 back2 back2 back2"
+      "back3 back3 back3 . . . project5 project5 project5 project5"
+      "back3 back3 back3 project6 project6 project6 project6  . back4 back4 "
+      "back3 back3 back3 . . . . . back4 back4";
+    margin-top: 120px;
+    gap: 0;
+  }
```

Comment:
- [ ] As requested by the previous reviewer, kindly ensure that your design matches the Figma template as much as possible. The projects grid is not properly arranged and some of the cards are stretched than they should, kindly fix this

| Your design | Figma template |
| ------------- | ------------- |
| ![Capture](https://user-images.githubusercontent.com/85095702/160215047-2ecf916e-73f9-4e6d-81fd-8a998bb06b1c.PNG) | ![figma](https://user-images.githubusercontent.com/85095702/160215087-dfe2c98a-42f7-4a3f-a106-c339e73d6cc1.PNG) |